cypress/views/tab.ts

```diff
@@ -1,19 +1,24 @@
-export const tabList = '.pf-c-tabs__list > li';
-
 export enum tabs {
   Configuration = 'horizontal-link-Configuration',
   Console = 'horizontal-link-Console',
   Diagnostics = 'horizontal-link-Diagnostics',
   Disks = 'horizontal-link-Disks',
   Environment = 'horizontal-link-Environment',
   Events = 'horizontal-link-Events',
+  Metadata = 'horizontal-link-Metadata',
   Metrics = 'horizontal-link-Metrics',
+  Migrations = 'horizontal-link-Migrations',
+  // customize creation tabs
   NetworkInterfaces = 'horizontal-link-Network interfaces',
+  // vm tabs
   Overview = 'horizontal-link-Overview',
-  Parameters = 'horizontal-link-Parameters',
+  Parameters = 'horizontal-link-Parameters', // template tab
   Scheduling = 'horizontal-link-Scheduling',
   Scripts = 'horizontal-link-Scripts',
+  Settings = 'horizontal-link-Settings',
   Snapshots = 'horizontal-link-Snapshots',
+  // virt overview tabs
```

Comment:
I think it will be more readable to split the tabs to multiple enums each representing a different group of tabs like detailsTabs, configurationSubTabs, overviewTabs and remove the comments that split those tabs

Reply:
split them into `vmTabs` and `overviewTabs`.